individual/services.py

```diff
@@ -111,6 +121,39 @@ def update_group_individuals(self, obj_data):
             return output_exception(model_name=self.OBJECT_TYPE.__name__, method="update", exception=exc)
 
 
+class CreateGroupAndMoveIndividualService(CreateCheckerLogicServiceMixin):
```

Comment:
I would like to see at least one integration test that steps through the happy path of this feature, just as a sanity/syntax test and as a documentation of how the feature works in code

Reply:
Test added